Cargo.toml

```diff
@@ -1,90 +1,90 @@
 [workspace]
 members = [
-    "crates/activity_indicator",
```

Comment:
Can you remove the whitespace spaces from the PR?

I realize we should probably check in a `.zed/settings.json` for our TOML formatting so that this doesn't happen automatically.

Reply:
I'll check this! Thanks

Reply:
@maxbrunsfeld I reverted those weird changes.